### ISSUE TITLE
Kodi 19 kompatibilitás

### ISSUE THREAD

**Issue by nistvan86** [OStreamH/repository.onlinestream#3]
Android TV esetében a Kodi frissült a legfrissebb 19 Matrix változatra, viszont ez nem kompatibilis az add-on-nal.

**Comment by ghost** [comment]
Szintén örülnék, ha elérhető lenne Kodi 19-re is :)

**Comment by brollyka** [comment]
A Kodi hivatalos oldalan hozza lehet ferni minden regi valtozathoz is, igy siman telepitheto barmire a 18.9 es verzio, amivel mukodik minden tokeletesen.

**Comment by ghost** [comment]
> A Kodi hivatalos oldalan hozza lehet ferni minden regi valtozathoz is, igy siman telepitheto barmire a 18.9 es verzio, amivel mukodik minden tokeletesen.

Persze, hogy mindent is meg lehet oldani, de ez egy hibajegy, mégpedig, hogy a kodi 19-el már nem kompatibilis, mert az már python3 alapú. Oké, hogy most még lehet használni a régit, de szépen lassan el fog teljesen avulni a 18.X verzió ezért lenne jó, ha a fejlesztők tudnák és esetleg frissítenék is a kódot :)
